### PR TITLE
translate using-the-state-hook

### PR DESCRIPTION
**Pull request by galnir**

### CODE REVIEW

File: content/docs/hooks-state.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 id: hooks-state
-title: Using the State Hook
+title: שימוש ב- State Hook
 permalink: docs/hooks-state.html
 next: hooks-effect.html
 prev: hooks-overview.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+*Hooks* הם תוסף חדש בריאקט 16.8. הם מאפשרים שימוש ב- state ופיצ'רים אחרים של ריאקט מבלי לכתוב מחלקה.
```
Comment (itayyehezkel) [inline content/docs/hooks-state.md]: Our code style says that we follow React in English.

File: content/docs/hooks-state.md [hunk 2]
```diff
@@ -28,11 +29,11 @@ function Example() {
...
 ```
 
-We'll start learning about Hooks by comparing this code to an equivalent class example.
+נתחיל ללמוד על Hooks על ידי השווה בין קוד זה לקוד המקביל שמשתמש במחלקה.
 
-## Equivalent Class Example {#equivalent-class-example}
+## דוגמת קוד מקביל {#equivalent-class-example}
 
-If you used classes in React before, this code should look familiar:
+אם השתמשת במחלקות בריאקט בעבר, קוד זה אמור להיראות מוכר:
```
Comment (itayyehezkel) [inline content/docs/hooks-state.md]: Here too, React should be in English.

File: content/docs/hooks-state.md [hunk 3]
```diff
@@ -28,11 +29,11 @@ function Example() {
 }
 ```
 
-We'll start learning about Hooks by comparing this code to an equivalent class example.
+נתחיל ללמוד על Hooks על ידי השווה בין קוד זה לקוד המקביל שמשתמש במחלקה.
 
-## Equivalent Class Example {#equivalent-class-example}
+## דוגמת קוד מקביל {#equivalent-class-example}
```
Comment (itayyehezkel) [inline content/docs/hooks-state.md]: What do you think about "דוגמה מקבילה במחלקה", I just try to stick to the original.

File: content/docs/hooks-state.md [hunk 4]
```diff
@@ -56,19 +57,19 @@ class Example extends React.Component {
...
+>הערה
 >
->You might be wondering why we're using a counter here instead of a more realistic example. This is to help us focus on the API while we're still making our first steps with Hooks.
+>אתה בטח תוהה למה אנו משתמשים ב- counter כאן במקום דוגמה יותר ריאליסטית. זה במטרה לעזור לנו להתמקד ב- API בזמן שאנחנו רק מתחילים עם Hooks.
 
-## Hooks and Function Components {#hooks-and-function-components}
+## Hooks וקומפוננטות פונקציונליות {#hooks-and-function-components}
 
-As a reminder, function components in React look like this:
+כתזכורת, קומפוננטות פונקציונליות בריאקט נראות כך:
```
Comment (itayyehezkel) [inline content/docs/hooks-state.md]: React should be in English.

File: content/docs/hooks-state.md (hunk 3 above)
Comment (galnir) [reply content/docs/hooks-state.md]: You might be right but imo "דוגמה מקבילה במחלקה" doesn't fit

**Comment by galnir** [comment]
Thanks @itayyehezkel, I took your suggestion and changed it to "דוגמה מקבילה עם שימוש במחלקה" instead of "דוגמה מקבילה במחלקה".

**Pull request merged**